src/open_inwoner/templates/pages/user-home.html

```diff
@@ -18,8 +18,12 @@ <h2 class="h2 h2--extra">
             {% render_card %}
                 <h3 class="h3">{{ plan.title }}</h3>
                 <p class="p">{{ plan.goal|truncatewords:20 }}</p>
-                <a class="button button--icon-before button--transparent button--secondary"
-                   href="{{ plan.get_absolute_url }}">
+                <a
+                    class="button button--icon-before button--transparent button--secondary"
+                    href="{{ plan.get_absolute_url }}"
+                    aria-describedby="{{ plan.title }}"
```

Comment:
aria-describedby should probably be aria-label, and an interactive element should not be made hidden to assitive technologies using aria-hidden as the latter describes more the availability of the item and not it's visual state.